docs/guide-to-triaging.md

```diff
@@ -0,0 +1,158 @@
+
+    Wowpedia often aggregates patch notes relating to a particular talent or ability, which is frequently useful. It also often records quest text and quest-giver dialogue, so can be a handy source for that sort of thing.
+
+- YouTube - can be an excellent source of information, as there are lots of videos of people doing quests in Classic/TBC. Sadly there are few from the original WotLK era. The other problem with YT is that retail players are not the only ones putting Wow videos up. Players on other private servers also make videos (watch out for one in particular called Bue, who will often be top result when searching for a quest on YouTube.) These private server videos are of course largely useless as sources for correct Blizz-like behaviour.
+
+- [TrinityCore](https://github.com/TrinityCore/TrinityCore/issues) - Trinity is older and larger than AC, and their own bug tracker and issues database is correspondingly larger than ours. As a result it is often a good place to check for other reports of an issue if the AC tracker doesn't have any results. You can also quite often find a solution for the issue as well. Be aware that TC has multiple branches, only one of which is targeting 3.3.5a.
+
+- Sniffs data - the new hotness. 'Sniffs' are intercepted data packets players have captured when playing retail Wow. They have the advantage of using real retail data and making it available in SQL, and thus broadly useable. The disadvantages are that the sniff data only contains client-visible data, it is often incomplete and patchy, and is often difficult to otherwise verify. 
+    For one useful site that makes use of this data, see the [AzerothCore SpeedChecker](https://azerothcore-speedchecker.web.app/) by @Efymer on Discord - if you want to verify a creature's movement speed or other flags versus sniffed retail data, this will do it for you.
+
+- [Wayback Machine/Internet Archive](https://web.archive.org/) - if you really get stuck for contemporary sources and need to see the state of a site in 2010, then Wayback Machine can be good, if you know the URL of the site you want to see. For example, they have a 2010 snapshot of the hunter pet site Petopia which has occasionally been useful.
+
+- Other forums (Wow/MMO-Champion) - the Wow forums can occasionally be useful when they discuss vanilla or TBC issues, but are usually full of people contradicting and flaming each other, so aren't very reliable as sources.
+
+- https://www.mmo-champion.com/forum.php - Similarly, MMO-Champion has forums that can be useful because they have been going, essentially unchanged, since before Wrath. This means you can sometimes find useful firsthand information there. However, like most forums, sorting the useful information from the random chaff can be difficult.
+
+- https://wowgaming.altervista.org/aowow/ - this is a site acting as the front end to a snapshot of our own AC database. As a result, it provides a rapid way of seeing
```

Comment:
Add a link, dont paste the full URL.

Is it even up-to-date?

The name for the website is also trash to represent that it should be azerothcore updated. But that's not on you :P

Reply:
Fixed the link, it's working so seems up-to-date. Will let Helias worry about the name. :)